pwndbg/commands/peda.py

```diff
@@ -19,15 +19,21 @@ def getfile() -> None:
     print(repr(pwndbg.auxv.get().AT_EXECFN))
 
 
-parser = argparse.ArgumentParser(description="Continue execution until an address or function.")
-parser.add_argument("target", type=str, help="Address or function to stop execution at")
+parser = argparse.ArgumentParser(
+    description="Continue execution until an address or function."
+)
+parser.add_argument("target", type=int, help="Address or function to stop execution at")
 
 
 @pwndbg.commands.ArgparsedCommand(parser, category=CommandCategory.NEXT)
 def xuntil(target) -> None:
+    running = pwndbg.gdblib.proc.alive
+    if not running:
```

Comment:
```suggestion
    if not pwndbg.gdblib.proc.alive:
```
No need to introduce a variable